<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Filename">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Filename"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_FirstRow">
       <property name="text">
        <string>First row</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_FirstRow">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_LastRow">
       <property name="text">
        <string>Last row</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_LastRow">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Qcolumn">
       <property name="text">
        <string>Q column</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_QColumn">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_RColumn">
       <property name="text">
        <string>R column</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_RColumn">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_EColumn">
       <property name="text">
        <string>Err column</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_EColumn">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>      Polarization</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>331</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_PolarizerPol">
         <property name="text">
          <string>Polarizer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Polx"/>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Poly"/>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Polz"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_AnalyzerPol">
         <property name="text">
          <string>Analyzer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Anax"/>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Anay"/>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Anaz"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="plaintexteditwithlinenumbers" name="plainTextEdit_FileContent">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plaintexteditwithlinenumbers</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">plaintexteditwithlinenumbers.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
